src/test/scala/zio/prelude/CommutativeEitherSpec.scala

```diff
@@ -11,10 +10,10 @@ object CommutativeEitherSpec extends DefaultRunnableSpec {
     testM("FutureCommutativeEither returns the first future that is completed") {
       for {
         l <- ZIO.fromFuture { implicit ec =>
-              Future.unit <|> Future.never
+              Future.successful("immediate") <|> Future { blocking { Thread.sleep(60 * 1000); "long 1" } }
             }
         r <- ZIO.fromFuture { implicit ec =>
-              Future.never <|> Future.unit
+              Future { blocking { Thread.sleep(60 * 1000); "long 2" } } <|> Future.successful("immediate")
```

Comment:
May be useful to define some implicit syntax here so these methods are available on a cross-version basis.